resources/js/Pages/Articles/Components/HighlightedArticles.tsx

```diff
@@ -78,11 +78,14 @@ export const HighlightedArticles = ({
 };
 
 const useArticlesSlidesPerPage = (): number => {
-    const { is2Xs, isXs, isSm, isMd, isMdLg } = useBreakpoint();
+    const { is2Xs, isXs, isSm, isMd, isMdLg, isMaxWidth } = useBreakpoint();
```

Comment:
kinda messy having this as it will be some sort of exception from the usual breakpoint usage 🤔 this also ignores the article columns, so those still switch from 2 to 3 at 768 while the featured articles switch at 769 due to this change.

don't change this yet, will first talk with Sam if this is what they really want or a design issue